core/impl-base/src/test/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoaderTestCase.java

```diff
@@ -44,7 +46,7 @@
    @Test
    public void shouldBeAbleToLoadAll() throws Exception
    {
-      ServiceRegistry registry = new ServiceRegistry(injector.get());
+      ServiceRegistry registry = new ServiceRegistry(injector.get(), new LinkedHashMap<Class<?>, Set<Class<?>>>());
```

Comment:
What you mention is done at `JavaSPIExtensionLoaderTestCase`.  This tests already existed before this change, it is just adapting to new contact. The idea is to not modify their purpose.